### DISCUSSION TITLE
Competition rules and use of external data

### DISCUSSION THREAD

**Discussion by eugene-NB** [PaddlePaddle/PaddleSpatial#8] [BaiduKDDCup2022] [answered]
Dear organizers,

What are the rules of the competition?
Are external data sources allowed?

Thanks in advance!

**Comment by zhoujingbo** [comment, marked as answer]
> Dear organizers,
> 
> What are the rules of the competition? Are external data sources allowed?
> 
> Thanks in advance!

Dear ;

The external data source is not allowed if without the approval of the challenge organizer. 
We have just updated the rules on the website: https://aistudio.baidu.com/aistudio/competition/detail/152/0/introduction

Thanks a lot.
Regards
Jingbo



> Rules

> Each submitted file should not exceed 200MB.

> Each team can submit up to 3 times per day (refreshed at 23:59 AOE).

> The awardees must submit the complete code and a complete report with technical details to reproduce the results by reading the report. The instructions for winning teams’ code and report submission will be released.

> No external data is allowed to use for this challenge, if without approval from the challenge organizers.

> Each participant is only permitted to be in ONE team. Any participant that is part of more than one Team may be disqualified and her/his corresponding Team may be disqualified, if the challenge organizers identify and/or are reported with evidence for such cases.

> Any participant who has a conflict of interest with Baidu Inc. and Longyuan Power Group (employees and partners, etc.) is not allowed to attend the challenge and is ineligible to win a prize. The right to verify such conflict of interests is reserved by the challenge organizers.

> The eligibility of each participant to the challenge should comply with the rules/policies by your employer/organization, the rules/policies by KDD Cup 2022, and the rules/policies by SIGKDD (ACM Special Interest Group on Knowledge Discovery and Data Mining). The challenge organizers reverse the right to verify such eligibility.

> The challenge organizers reserve the right to limit or restrict participation in the Challenge to any person at any time for any reason.

> A team will be disqualified for the award if 1) fails to submit the code and report before the deadline, or 2) the summitted code cannot reproduce the results, or 3) the technical report is not identical to the code, or 4) violated any other rules.

> The challenge organizers reserve the right to update the award setting, competition timeline, and rules.
